src/components/common/SimpleTable/index.tsx

```diff
@@ -50,19 +56,30 @@ const Wrapper = styled.table<{ $numColumns?: number }>`
     > tr {
       display: grid;
       width: 100%;
-      transition: background 0.1s ease-in-out;
-      border-bottom: 0.1rem solid var(--color-border);
+      transition: background-color 0.1s ease-in-out;
+      height: ${CELL_HEIGHT};
+      padding: 1rem;
+      box-sizing: border-box;
+
+      &:not(:last-of-type) {
+        border-bottom: 0.1rem solid ${TR_BORDER_COLOR};
+      }
+
+      @media ${MEDIA.mobile} {
```

Comment:
do we not have the `theme.upToSmall` for this?

Reply:
@W3stside Updated to use that, after help from @alfetopito . I do have some thoughts on that approach. The new approach:

```
      ${applyMediaStyles('upToSmall')`
        display: flex;
        flex-flow: column wrap;
        height: auto;
        align-items: flex-start;

        &:hover {
          background: var(--color-text-hover);
          > td {
            color: var(--color-text-primary);
          }
        }
      `}
```

Whereas before we had:
```
@media ${MEDIA.mobile} {
    ..CSS HERE..
}
```

In my view, the previous method was more elegant:
- Respect native (S)CSS. 
- Allows for normal syntax highlighting (because of native (S)CSS).
- Simple use of @media + a variable (could still come from our theme as it did).

The new approach is so much more verbose and harder to read/debug IMO, because of using a string literal for the code:
<img width="599" alt="Screen Shot 2021-02-09 at 17 12 11" src="https://user-images.githubusercontent.com/31534717/107392417-f41eeb00-6af9-11eb-9b1c-08179ddbc5e1.png">

Your call otherwise.